src/test/mod.rs

```diff
@@ -0,0 +1,151 @@
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+
+use anyhow::Context;
+use async_trait::async_trait;
+use hyper::body::Bytes;
+use reqwest::{Client, Request};
+
+use crate::cache::InMemoryCache;
+use crate::http::Response;
+use crate::target_runtime::TargetRuntime;
+use crate::{EnvIO, HttpIO};
+
+macro_rules! include_file {
+    ($name:literal) => {
+        include_str!(concat!(env!("CARGO_MANIFEST_DIR"), $name))
+    };
+}
+
+const NESTED0: &str = include_file!("/src/grpc/tests/nested0.proto");
+const NESTED1: &str = include_file!("/src/grpc/tests/nested1.proto");
+const GREETINGS: &str = include_file!("/src/grpc/tests/greetings.proto");
+const NEWS: &str = include_file!("/src/grpc/tests/news.proto");
+const CYCLE: &str = include_file!("/src/grpc/tests/cycle.proto");
+const DUPLICATE: &str = include_file!("/src/grpc/tests/duplicate.proto");
+const JSONPLACEHOLDER_JSON: &str = include_file!("/examples/jsonplaceholder.json");
+const JSONPLACEHOLDER_YML: &str = include_file!("/examples/jsonplaceholder.yml");
+const JSONPLACEHOLDER_GQL: &str = include_file!("/examples/jsonplaceholder.graphql");
+const JSONPLACEHOLDER_SCRIPT: &str = include_file!("/examples/jsonplaceholder_script.graphql");
+const ECHO_JS: &str = include_file!("/examples/scripts/echo.js");
```

Comment:
copy the files in `src/test/fs` and read those files here.

Reply:
Create a const of `HashMap<FileName, FileContent>` instead of consts for each file.